examples/matrix-docker-ansible-deploy/vars.yml

```diff
@@ -0,0 +1,209 @@
+# SPDX-License-Identifier: 2023 - 2025 Slavi Pantaleev
+# SPDX-License-Identifier: 2023 Julian-Samuel Gebühr
+# SPDX-License-Identifier: 2025 Suguru Hirahara
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+---
```

Comment:
The file path `examples/matrix-docker-ansible-deploy/vars.yml` makes me think that this file contains `matrix-docker-ansible-deploy` configuration that I (as a MASH user) should use to set up Matrix.

At least, I suppose many could understand it this way.

I wonder if renaming the `matrix-docker-ansible-deploy` directory to `mash-for-matrix-docker-ansible-deploy-users` would make it clearer?

Reply:
That surely makes sense! :smile:

Reply:
Addressed with https://github.com/mother-of-all-self-hosting/mash-playbook/pull/390/commits/efd24ec1152ae6c84167c378b5209be853a5f998.